data-access/UserRepository.js

```diff
@@ -0,0 +1,53 @@
+export default class UserRepository {
+    constructor(models, Op) {
+        this.models = models;
```

Comment:
It's not the best approach to pass all the models into a module that supposed to manage only one model